tools/scripts/launch-wisdom.sh

```diff
@@ -29,6 +29,7 @@ if [ "${DEPLOYMENT_MODE}" == "upstream" ]; then
     # for upstream, creating a out-of-box tesing user for starting quick
     echo "Creating a testing user for upstream mode..."
     /var/www/venv/bin/wisdom-manage createtoken --username testuser --password testuser --token-name testuser_token --create-user
+	/var/www/venv/bin/wisdom-manage createsuperuser --noinput --username admin --email admin@example.com
```

Comment:
This shouldn't be here. This will result in this admin superuser being created in production.